Python-packages/covidcast-py/covidcast/plotting.py

```diff
@@ -74,39 +79,51 @@ def get_geo_df(data: pd.DataFrame,
-    type. By default, it will take the input data (left side) and geo data (right side) and right
-    join them, so all states/counties will always be present regardless whether ``data`` contains
-    values for those locations. ``left``, ``outer``, and ``inner`` joins are also supported and
-    can be selected with the ``join_type`` argument.
-
-    For right joins on counties, all counties without a signal value will be given the value of
-    the megacounty (if present). Other joins will not use megacounties.
-
-    Returns the columns from ``data`` along with
-    `geometry` (polygon for plotting) and `state_fips` (FIPS code which will be used in the
-    plotting function to rearrange AK and HI) column. Coordinate system is GCS NAD83.
-
-    Default arguments for column names correspond to those used by :py:func:`covidcast.signal`.
-    Currently only supports counties and states.
+    """Augment a :py:func:`covidcast.signal` data frame with the shape of each geography.
+
+    This method takes in a pandas DataFrame object and returns a GeoDataFrame
+    object from the `GeoPandas package <https://geopandas.org/>`_. The
+    GeoDataFrame will contain the geographic shape corresponding to every row in
+    its ``geometry`` colummn; for example, a data frame of county-level signal
+    observations will be returned with the shape of every county.
+
+    After detecting the geography type (only county and state are currently
+    supported) of the input, this function builds a GeoDataFrame that contains
+    state and geometry information from the Census for that geography type. By
+    default, it will take the signal data (left side) and geo data (right side)
+    and right join them, so all states/counties will always be present
+    regardless whether ``data`` contains values for those locations. ``left``,
+    ``outer``, and ``inner`` joins are also supported and can be selected with
+    the ``join_type`` argument.
+
+    For right joins on counties, all counties without a signal value will be
+    given the value of the megacounty (if present). Other joins will not use
+    megacounties. See the `geographic coding documentation
+    <https://cmu-delphi.github.io/delphi-epidata/api/covidcast_geography.html>`_
+    for information about megacounties.
+
+    By default, this function identifies the geography for each row of the input
+    data frame using its ``geo_value`` column, matching data frames returned by
+    :py:func:`covidcast.signal`, but the ``geo_value_col`` and ``geo_type_col``
+    arguments can be provided to match geographies for data frames with
+    different column names.
+
+    Geographic data is sourced from 1:5,000,000-scale shapefiles from the `2019
+    US Census Cartographic Boundary Files
+    <https://www.census.gov/geographies/mapping-files/time-series/geo/cartographic-boundary.html>`_.
 
     :param data: DataFrame of values and geographies
```

Comment:
```suggestion
    :param data: DataFrame of values and geographies.
```
Thought i had fixed these in a past PR but maybe got mixed up in merge conflict.